skunkworks_crow/src/main/res/values/strings.xml

```diff
@@ -131,6 +131,10 @@
     <string name="odk_destination_dir_error">The destination should not be empty</string>
     <string name="location_permission_needed">Location Permission Needed</string>
     <string name="location_settings_dialog">Enable location from the settings.</string>
+    <string name="reset_settings">Click to reset all preferences</string>
```

Comment:
This is not required, the title gives enough context about the action that is supposed to happen when user taps on it.